android/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
     ext {
         buildToolsVersion = "30.0.3"
         minSdkVersion = 24
-        compileSdkVersion = 30
+        compileSdkVersion = 31
         targetSdkVersion = 30
```

Comment:
Can we increase targetSdkVersion as well? 
I have Android 12 (31 API), works okay for me